src/test/java/io/github/wimdeblauwe/hsbt/mvc/MyHtmxResponse.java

```diff
@@ -0,0 +1,19 @@
+package io.github.wimdeblauwe.hsbt.mvc;
+
+import org.springframework.ui.Model;
+
+public class MyHtmxResponse extends HtmxResponse {
```

Comment:
I'd like to find another way for this. I don't really like subclassing just for code re-use purposes. I would find it nicer if this was modeled as a factory method, something like:

```
public final class MyApplicationHtmxResponseHelper {
  public HtmxResponse sendAlertPartial(Model model, String alertText) {
    HtmxResponse response = new HtmxResponse();
    response.addTemplate("users :: alert");
    model.addAttribute("alertText", alertText);
    response.addTrigger("alertSent");
    }
}
```

This would allow us to make `HtmxResponse` final. And maybe even immutable if we would create a `HtmxResponse.Builder` class.

Reply:
That suits me. I'll make `HtmxResponse` final and add the ability merge with another `HtmxResponse`